olive/passes/onnx/genai_model_exporter.py

```diff
@@ -54,17 +78,12 @@ def _run_for_config(
         output_model_filepath = Path(resolve_onnx_path(output_model_path))
 
         precision = config["precision"]
-        device = (
-            Device.CPU
+        execution_provider = (
+            "cpu"
             if self.accelerator_spec.execution_provider
```

Comment:
>self.accelerator_spec.execution_provider [](http://example.com/codeflow?start=15&length=40)

host_device or target_device?

Reply:
Pass is in any device. Model is in target device.